use_case_examples/llm/QGPT2Evaluate.ipynb

```diff
@@ -270,24 +271,54 @@
     "output_logits_simulated = proj_single_head_qgpt2(input_ids).logits"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we are ready to execute the single head attention in FHE"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 14,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Simulated logits are equal to the quantized clear ones: False\n"
+      "Time taken to execute the single head in FHE: 166.38 seconds\n"
```

Comment:
Yeah with InsertRounding we aren't equal anymore, best to show FHE runtime

Reply:
We aren't equal to disable (expected) but there seem to be a different between simulate and fhe. Would that be expected?